tools/vipps-partner-api-postman-environment.json

```diff
@@ -4,23 +4,23 @@
 	"values": [
 		{
 			"key": "client_id",
-			"value": "<Use the partner keys you have received>",
+			"value": "<Go To Vipps Portal->Select the Developer tab->Select Test or Production Keys->Select 'Show keys'->copy the Client Id here>",
 			"type": "default",
 			"enabled": true
 		},
 		{
 			"key": "client_secret",
-			"value": "<Use the partner keys you have received>",
+			"value": "<Go To Vipps Portal->Select the Developer tab->Select Test or Production Keys->Select 'Show keys'->copy the Client secret here>",
 			"enabled": true
 		},
 		{
 			"key": "Ocp-Apim-Subscription-Key",
-			"value": "<Use the partner keys you have received>",
+			"value": "<Go To Vipps Portal->Select the Developer tab->Select Test or Production Keys->Select 'Show keys'->copy the Vipps-Subscription-Key>",
 			"enabled": true
 		},
 		{
 			"key": "merchantSerialNumber",
-			"value": "<Use the merchantSerialNumber from getMerchan or from registered sales unit>",
+			"value": "<Go To Vipps Portal->Select the Developer tab->Select Test or Production Keys->Copy the Merchant Serial Number from the Sale Unit you copied the keys from>",
 			"enabled": true
```

Comment:
This is not not really wrong, but the partner doesn't necessarily have their own sales merchant that they can do this for. It's not their own sales unit they need, but the one of the merchant registered with them as a partner.

Reply:
I will revert it back to the previous definition. I'm not sure what "getMerchan" is. Is this correct?

Reply:
getMerchant is just one of the endpoints, so I'm not sure if that part is needed, because you probably already have it at that point.

Reply:
"getMerchant" is probably the operation id? We don't have to be that specific. We can probably just say "Use the merchant's MSN (merchant serial number)". The partner must be able to figure this out.

Reply:
updated